lib/cylc/task_pool.py

```diff
@@ -822,7 +822,9 @@ def match_dependencies(self):
         """
         all_outputs = {}   # all_outputs[message] = taskid
```

Comment:
Sorry, this is not possible at the moment, as it has implication all the way down the `satisfy_me` method of the prerequisite objects.